server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableImpl.java

```diff
@@ -262,7 +268,7 @@ void finishSelection(Set<StoredTabletFile> selected, boolean allSelected) {
       Preconditions.checkArgument(!selected.isEmpty());
       Preconditions.checkState(selectStatus == FileSelectionStatus.SELECTING);
       selectStatus = FileSelectionStatus.SELECTED;
-      selectedTime = System.currentTimeMillis();
+      selectedTime = getCurentTimeMillis();
```

Comment:
This method is spelled wrong. Also, in order to avoid timing bugs with system clock changes, which can occur at any time in the system for many reasons (NTP adjustments, daylight savings adjustments, user switching the time zone or setting the clock manually, etc.), this should really be based on nanoTime if all the times are retrieved from the same running process.

Reply:
It would be better to use nano time.  Use of currentTimeMillis was present before this change.  I may update it in this change, if not we can open an issue.

Reply:
@ctubbsii I updated to use nanoTime in 255ba3754d4686ebc903b1ef2e135b4ef8904598.  Do those changes look correct?  The unit test does not actually call the nanoTime system call so if I botched something w/ the update this test I am adding would not catch it.

Reply:
It looks right to me.

Reply:
> It looks right to me.

Thanks.  The main thing I was concerned about was the conversion of the duration from accumuo config in millis to nanos (this is done in the deriver).  I looked at that code multiple times and was 99% sure it was correct, but still felt it would be nice to have another set of eyes on it.

Reply:
@ctubbsii  I updated the code to use the Duration type.  I think this is cleaner and a bit less error prone.  I also looked into using the Instant type in the code, but its not a good fit for nanoTime.

Thinking it may be nice to add a `Duration getDuration(Property)` method to AccumuloConfiguration, but not sure.